steps/android-info/1.0.1/step.yml

```diff
@@ -0,0 +1,51 @@
+title: Android info
+summary: Android APK info provider
+description: Provides all possible Android APK information as package name, version
+  name or version code.
+website: https://github.com/thefuntasty/bitrise-step-android-info
+source_code_url: https://github.com/thefuntasty/bitrise-step-android-info
+support_url: https://github.com/thefuntasty/bitrise-step-android-info/issues
+published_at: 2016-10-19T14:14:42.059481445+02:00
+source:
+  git: https://github.com/thefuntasty/bitrise-step-android-info.git
+  commit: 9d343a42df2b8a3b64ec328b236f1fc609aaf0d8
+project_type_tags:
+- android
+type_tags:
+- artifact
+is_requires_admin_user: false
+is_always_run: true
```

Comment:
I'm not sure about this - if a previous step failed I'd say there is no advantage of running this step